docs/reference/cat/snapshots.asciidoc

```diff
@@ -12,7 +12,7 @@ consumption, use the <<get-snapshot-api,get snapshot API>>.
 ====
 
 Returns information about the <<snapshot-restore,snapshots>> stored in one or
-more repositories. A snapshot is a backup of an index or running {es} cluster.
+more <<snapshots-register-repository,repositories>>. A snapshot is a backup of an index or running {es} cluster.
```

Comment:
I would skip this link, because the core snapshot docs introduce the concept of repositories.

```suggestion
more repositories. A snapshot is a backup of an index or running {es} cluster.
```